tests/e2e/onboarding.test.ts

```diff
@@ -131,6 +140,134 @@ test('onboarding with a short code', async ({ page, getOnboardingData }) => {
 	await expect(page).toHaveURL(`/onboarding`)
 })
 
+test('onboarding with GitHub OAuth', async ({ page }) => {
+	const code = `${MOCK_CODE_GITHUB}_${crypto.randomUUID()}_PLAYWRIGHT_TEST`
+
+	await page.route(/\/auth\/github(?!\/callback)/, async (route, request) => {
```

Comment:
It's perfectly fine as it is but since Epic Stack uses MSW already, we may take advantage of the shared network description between integration and E2E tests. 

There are a few request handlers defined for GitHub auth already:
https://github.com/epicweb-dev/epic-stack/blob/800022dae82530dd48ef0ba35abe666eac1c619e/tests/mocks/github.ts#L124-L125

In the future, we may refactor this and replace `page.route()` with the [browser-side integration](https://mswjs.io/docs/integrations/browser) of MSW to intercept this request and produce whichever response is needed (e.g. the one including a proper `code`).

Reply:
It's just so tempting to use such a clean API :)

Reply:
Yeah, it's perfectly fine!